src/pfs/pfs.proto

```diff
@@ -106,13 +107,6 @@ message Commit {
   Branch branch = 2;
 }
 
-// CommitRange represents chain of commits with Lower being an ancestor of
-// Upper or, in the case of a range of size 1, the same commit.
-message CommitRange {
-  Commit lower = 1;
-  Commit upper = 2;
-}
-
 // CommitProvenance keeps track of where (i.e. which branch) a certain commit
```

Comment:
Do we still need this?

Reply:
Also, some of the comments in this file need to be updated.

Reply:
We can totally get rid of this and it's on my list, but I didn't want to bloat this PR (hah).